regolith/helpers/l_abstracthelper.py

```diff
@@ -22,19 +23,26 @@ def subparser(subpi):
     subpi.add_argument(
         "-a",
         "--author",
-        help='authors group ID(single argument only) to use to find presentation abstract.')
+        help='authors group ID(single argument only) to use to find '
+             'presentation abstract.')
     subpi.add_argument(
         "-y",
         "--year",
-        help='Start or end year of the presentation (single argument only) to use to find presentation.')
+        help='Start or end year of the presentation (single argument only) to '
+             'use to find presentation.')
     subpi.add_argument(
         "-l",
-        "--location",
-        help='Location of presentation, either a country, city, state, or university.')
+        "--loc_inst",
+        help='Location of presentation, either a fragement of an institution, '
+             'country, city, state, or university. If an instiution is entered,'
+             'the search will be for seminars or colloquiums, otherwise the '
+             'search will be for meetings')
+
```

Comment:
disregard the empty line at 40, it is actually removed (slipped my gaze initially) but I will push that in the next big change